roles/influxdb-retriever/templates/influxdb_retriever_deployment.yaml.j2

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    name: influxdb-retriever
+    app: dojot
+  name: influxdb-retriever
+  namespace: {{ dojot_namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: dojot
+      name: influxdb-retriever
+  template:
+    metadata:
```

Comment:
This service supports livenessProbe, readinessProbe and startupProbe. It doesn't necessarily make sense to put all 3, but...
The default port is 9000, see more at https://github.com/gajus/lightship/

Reply:
I think at least livenessProbe makes sense and maybe readinessProbe too.

Reply:
OK, very good!

Reply:
That was changed.